lib/Base/Tensor.cpp

```diff
@@ -219,8 +220,8 @@ template <class ElemTy>
 static bool tryTransposeFastImpl(Handle<ElemTy> &src, Handle<ElemTy> &dest,
                                  llvm::ArrayRef<unsigned> shuffle) {
   const size_t numDims = dest.dims().size();
-  size_t srcCoorArr[numDims];
```

Comment:
GLOW_VLA is not very readable. I prefer to just use array of size "max_tensor_dimensions" here.